### ISSUE TITLE
Array key values are not displaying correctly

### ISSUE THREAD

**Issue by deanhouseholder** [kint-php/kint#362]
I've been using Kint for years and love it! I have never noticed this problem before.

I've got a plain array where the array key values are not being displayed correctly.

```php
d($users);
print "<pre>\n";
print_r($users);
print "</pre>\n";
```

![Kint_bug_no_array_keys](https://user-images.githubusercontent.com/717841/121071225-e833f480-c78c-11eb-8ead-c630caac8dd4.png)

This is happening on Kint version 3.3 (currently the latest release).

**Comment by jnvsor** [comment]
That's master, elision isn't in 3.x

Also that shouldn't be happening, so I'll fix it in master and get back to you

**Comment by jnvsor** [comment]
Hi @deanhouseholder - this should be fixed in 913ca16353107dcc47a78361f03c965bb9e10711 if you download a new phar from the site

**Comment by deanhouseholder** [comment]
That fixed it. Thanks!
